doc/cli-directive.md

```diff
@@ -215,6 +222,16 @@ cli:
           op: CreateOrUpdate#Update
           param: properties
         cli-flatten: true
+    # set example content with static value
+      - where:
+          exampleName: TheExampleName
+          examplePath: parameters.hostPool.location
```

Comment:
Should the example path be the exampleParameterPath ? It's a little confusing

Reply:
yes, committed.